torch/csrc/jit/autodiff.cpp

```diff
@@ -38,6 +38,7 @@ bool needTrimGrad(Node* n) {
       "aten::kthvalue(Tensor self, int k, int dim, bool keepdim) -> (Tensor, Tensor)",
```

Comment:
IIRC this was not needed any more, worth trying to delete it in a follow up PR. ;)

Reply:
I created T52707688 for this.